containers/ecr-viewer/integration/saveFhirDataService/core.test.ts

```diff
@@ -103,6 +139,40 @@ describe("saveFhirData - core", () => {
     expect(resp.status).toEqual(200);
   });
 
+  it("should reference the condition_code foreign key", async () => {
+    const db = getDb<Core>();
+    const metadata: BundleMetadata = {
+      ...baseCoreMetadata,
+      rr: [
+        {
+          condition: "flu",
+          code: "123",
+          rule_summaries: [{ summary: "fever" }, { summary: "influenza" }],
+        },
+      ],
+    };
+
+    const resp = await saveFhirMetadata(
+      "1-2-3-4",
+      "core",
+      metadata,
+      makePromiseResolveWithStatus(200),
+      () => {
+        return makePromiseResolveWithStatus(200);
+      },
+    );
+
+    const conditions = await db
+      .selectFrom("ecr_rr_conditions")
+      .selectAll()
+      .execute();
+
+    expect(resp.message).toEqual("Success. Saved metadata to database.");
+    expect(resp.status).toEqual(200);
+    expect(conditions).toHaveLength(1);
+    expect(conditions[0].condition_code).toEqual("123");
```

Comment:
given there's not a true fk anymore, do we need the setup with the user and program areas?